session/common.go

```diff
@@ -308,6 +308,16 @@ func (x *commonData) SetAuthKey(key neofscrypto.PublicKey) {
 	x.authKey = x.authKey[:key.Encode(x.authKey)]
 }
 
+// SetIssuer allows to set issuer before sign call.
```

Comment:
Please also note that it shouldn't be used normally and if you're using that it's your responsibility to ensure it matches the ID of the signer.